wad_project/templates/socshare/profile.html

```diff
@@ -7,6 +7,10 @@
     <img src="{{ MEDIA_URL }}{{ logo }}" class="logo" alt="">
     <h3 class="logo-name">{{ name }}</h3>
 </div>
+<br>
+<center><h2> Events </h2></center>
```

Comment:
Why?

Reply:
There needs to be more information added into the profile. So thats a separator :)